Rakefile

```diff
@@ -1,14 +1,13 @@
 require 'bundler/gem_tasks'
-require 'rake/testtask'
-require 'rubocop/rake_task'
 
-RuboCop::RakeTask.new
+begin
+  require 'rspec/core/rake_task'
+  require 'rubocop/rake_task'
 
-task default: :test
+  RuboCop::RakeTask.new
+  RSpec::Core::RakeTask.new(:spec)
 
-Rake::TestTask.new do |t|
-  t.verbose = true
-  t.libs    = ['lib']
-  t.libs.push 'test'
-  t.test_files = FileList['test/**/*_test.rb']
+  task default: :spec
+rescue LoadError
```

Comment:
Yeah, it's just the recommended practice from the RSpec devs (it allows rake to be used in non development environments). We don't actually use rake outside of development so I don't think it's particularly important to keep around.